pkg/clustermesh/clustermesh.go

```diff
@@ -37,15 +38,12 @@ const (
 // Configuration is the configuration that must be provided to
 // NewClusterMesh()
 type Configuration struct {
```

Comment:
Extremely minor nit: "Configuration" is a confusing name. It's really a set of dependencies, not a configuration.

But, like, it's also very much not a big deal.

Reply:
That's right that it is not a pure configuration anymore. I'm not sure about a better name though, *Dependencies* also looks a bit weird when accessing for instance the cluster name as `rc.mesh.deps.ClusterName` (I personally feel that `rc.mesh.conf.ClusterName`  is a bit more natural here).